seaborn/rcmod.py

```diff
@@ -540,12 +533,9 @@ def set_palette(palette, n_colors=None, desat=None, color_codes=False):
 
     """
     colors = palettes.color_palette(palette, n_colors, desat)
-    if mpl_ge_150:
-        from cycler import cycler
-        cyl = cycler('color', colors)
-        mpl.rcParams['axes.prop_cycle'] = cyl
-    else:
-        mpl.rcParams["axes.color_cycle"] = list(colors)
+    from cycler import cycler
```

Comment:
Since this doesn't need to be under a conditional, pleas move it to the top of the module.

Reply:
Thanks, updated